django_project/django_project/config.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+class AppConfig:
```

Comment:
nit: I think there is no need to create own file for config, envs can be used directly in settings.py